extension/src/test/e2e/extension.test.ts

```diff
@@ -143,7 +145,7 @@ describe('Plots Webview', function () {
 
     await browser.waitUntil(
       async () => {
-        return (await webview.vegaVisualization$$.length) === 5
+        return (await webview.vegaVisualization$$.length) === 6
```

Comment:
On a 2nd local run I think this will be 7 then on the 3rd 8. Need to remove the plot even if the test fails.

Reply:
I am probably wrong. I still would not rely on that behaviour. Image if it changed, how hard/confusing would it be to debug the test?